plugins/by-name/coverage/default.nix

```diff
@@ -183,13 +183,13 @@ in
     '';
 
     summary = {
-      widthPercentage = helpers.defaultNullOpts.mkNullable (types.numbers.between 0.0
-        1.0
-      ) 0.7 "Width of the pop-up window.";
+      widthPercentage =
+        helpers.defaultNullOpts.mkNullable (types.numbers.between 0.0 1.0) 0.7
+          "Width of the pop-up window.";
 
-      heightPercentage = helpers.defaultNullOpts.mkNullable (types.numbers.between 0.0
-        1.0
-      ) 0.5 "Height of the pop-up window.";
+      heightPercentage =
+        helpers.defaultNullOpts.mkNullable (types.numbers.between 0.0 1.0) 0.5
+          "Height of the pop-up window.";
```

Comment:
This seems better. Change comes from [NixOS/nixfmt@`64d3540` (#269)](https://github.com/NixOS/nixfmt/pull/269/commits/64d35408ab17a3c1b18b4a0cfef250debdb994c8) ([nixpkgs diff](https://github.com/infinixbot/nixpkgs/commit/515383e2ac74dd3b464928d4abc7300ff2be5990))